docs/contributing/coding-guidelines/index.md

```diff
@@ -14,6 +14,58 @@ Also, keep in mind the following concepts.
 
 - Keep things consistent.
 - Automate where possible, using simple checks for formatting, syntax, etc.
+- Write comments and documentation in English.
+- Functions that are too complex (low cohesion) should be appropriately split into smaller functions.
+- Try to minimize the use of member variables or global variables that have a large scope.
+- Whenever possible, break large pull requests into smaller, manageable PRs.
 - When it comes to code reviews, don't spend too much time on trivial disagreements. For details see:
   - <https://en.wikipedia.org/wiki/Law_of_triviality>
   - <https://steemit.com/programming/@emrebeyler/code-reviews-and-parkinson-s-law-of-triviality>
+
+## C++ Style Guide
+
+As a basic rule, follow [the Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html) and [the C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md). The following are common areas requiring special attention:
```

Comment:
Maybe we could also refer to C++ Coding guideline page in Autoware Documentation as well:
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/

Reply:
Thank you!! I updated in https://github.com/autowarefoundation/autoware-documentation/pull/599/commits/0d2d9c5fa6784d29b45e153ab1b0b17c930293a9